src/components/victory-line.jsx

```diff
@@ -223,21 +223,15 @@ class VictoryLine extends React.Component {
 
   render() {
     if (this.props.animate) {
-      // dont interpolate y if it is a function!
-      const yFunc = _.isFunction(this.props.y) ? this.props.y : undefined;
```

Comment:
Note that I removed the exception for `y` because we actually can interpolate functions now! :)